pkg/config/config.go

```diff
@@ -102,3 +111,59 @@ func (c *Config) getVfInfo(vfPci string) (string, int, error) {
 
 	return pf, vfID, nil
 }
+
+// handleBridgeConfig checks CNI bridge configuration and set ActualBridge options for PluginConfig.
+// If config.Bridge option is empty, config.ActualBridge will be the value of DefaultBridge const.
+// If config.Bridge option contains one bridge name, config.ActualBridge will be that bridge.
+// If config.Bridge option contains a list of bridges, then auto-detect logic will be used,
+// and the function will iterate over configured bridges and check to which bridge uplink is connected.
+// Uplink should be added to one of the bridges when multiple bridges are set in config,
+// this is required for auto-detect logic.
+// When only one bridge is configured there will be no validation that uplink is a part of a bridge,
+// this is required for backward compatibility.
+func (c *Config) handleBridgeConfig(conf *localtypes.PluginConf) error {
+	if conf.Bridge == "" {
+		conf.Bridge = DefaultBridge
+	}
+	bridgeNamesInConf := strings.Split(conf.Bridge, ",")
+	allowedBridgeNames := make([]string, 0, len(bridgeNamesInConf))
+	for _, brName := range bridgeNamesInConf {
+		brName = strings.TrimSpace(brName)
+		if brName != "" {
+			allowedBridgeNames = append(allowedBridgeNames, brName)
```

Comment:
so if bridge == "foo,," you will just skip it. why we don't return that 
"bridge configuration option has invalid format" ?

Reply:
In the code version to which you added the comment, behavior was the following: "foo,," > use "foo" as a single value, ignore ",,".

I changed validation to be more strict "foo,"  "foo,,"  ",foo"   - will return error instead of silently using "foo" as a single value